.github/workflows/pr_build.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Pull Request Build, Tag, & Push
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+      - master
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
```

Comment:
Why do builds on a close?

Reply:
Not sure, maybe it should be "merged" instead?

Reply:
I'm just updating active repos to the lastest version of the DevOps scripts by pressing the button to add workflows. It works on closed, but the goal is to get rid of these old scripts and use the new scripts on repos that use the new actions.

Reply:
I think I got confused, “merged” is not a valid option.. So yea close is the only way to trigger a build on merge I think I got confused, “merged” is not a valid option